manifests/kustomize/README.md

```diff
@@ -51,6 +51,13 @@ Its storage is based on CloudSQL & GCS. It's better than others for production u
 
 Please following [sample](sample/README.md) for a customized installation.
 
+### Option-4 Install it to AWS with S3 and RDS MySQL
+Its storage is based on S3 & AWS RDS. It's more natural for AWS users to use this option.
+
+Please following [AWS Instructions](env/aws/README.md) for installation.
+
+Note: Community maintains a repo [here](https://github.com/e2fyi/kubeflow-aws/tree/master/pipelines) for AWS.
```

Comment:
Change `here` to `e2fyi/kubeflow-aws` for visibility.

Reply:
This is duplicated with https://github.com/kubeflow/pipelines/blame/dcf69dacd25cd8e104010c62d2cb0d6ea5c1fe3c/manifests/kustomize/README.md#L11

Please remove above line.

Reply:
Fixed

Reply:
my bad. my point is to keep `Note: Community maintains a repo [here](https://github.com/e2fyi/kubeflow-aws/tree/master/pipelines) for AWS.`  and remove #11.